src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -498,7 +498,7 @@ function ComposerWithSuggestions(
      */
     const replaceSelectionWithText = useCallback(
         (text: string) => {
-            updateComment(ComposerUtils.insertText(commentRef.current, selection, text));
+            updateComment(ComposerUtils.insertText(commentRef.current, selection, text), true);
```

Comment:
NAB, can you please add a comment to explain why we should use debounce here?

Reply:
yes, i will do it later today